ncs_con/cam_msg_2d[] cams